kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -213,6 +212,14 @@
     height: 35px
```

Comment:
@DXCanas Tota11y :cool: with me, as long as the `:focus` states are implemented. This for example looks fine to me for search input field:
```
  input[type='search']
  // snip
    &:focus
      outline: none
      border: solid 1px
      border-color: $core-bg-light
      box-shadow: 0 0 5px 1px rgba(114, 72, 111, 0.75)
```
And this would be a good for the drop-down:

```
  .user-filter-dropdown
   // snip
    &:focus
      outline: none
      box-shadow: 0 0 5px 2px rgba(114, 72, 111, 0.75) inset

  option[selected]
    background-color: $core-action-light
```
Unfortunately no way to change the default red of selected option without JS, so let's leave it at that.